sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/models/MetricsBatchQueryOptions.java

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.query.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.Duration;
+import java.util.List;
+
+/**
+ * The model class to configure the metrics batch query options.
+ */
+@Fluent
+public final class MetricsBatchQueryOptions {
```

Comment:
The description of the PR says this was added to make the namespace required but the design here doesn't enforce that and that is still left to the APIs taking this type to check `namespace != null`

Reply:
This type doesn't have `namespace` property. This is directly on the method API which enforces the param to be set.